x-pack/plugins/maps/public/classes/layers/new_vector_layer_wizard/config.tsx

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { i18n } from '@kbn/i18n';
+import React from 'react';
+import { LayerWizard, RenderWizardArguments } from '../../layers/layer_wizard_registry';
+import { NewVectorLayerEditor } from './index';
+import { DrawLayerIcon } from '../../layers/icons/draw_layer_icon';
+import { CREATE_DRAWN_FEATURES_INDEX_STEP_ID, ADD_DRAWN_FEATURES_TO_INDEX_STEP_ID } from './wizard';
+import { getFileUpload } from '../../../kibana_services';
+
+export const newVectorLayerWizardConfig: LayerWizard = {
+  categories: [],
+  description: i18n.translate('xpack.maps.newVectorLayerWizard.description', {
+    defaultMessage: 'Draw points & shapes and save in Elasticsearch',
+  }),
+  disabledReason: i18n.translate('xpack.maps.newVectorLayerWizard.disabledDesc', {
+    defaultMessage:
+      'Unable to draw vector shapes, you are missing the Kibana privilege "Index Pattern Management".',
+  }),
+  getIsDisabled: async () => {
+    const hasImportPermission = await getFileUpload().hasImportPermission({
+      checkCreateIndexPattern: true,
+      checkHasManagePipeline: false,
+    });
+    return !hasImportPermission;
+  },
+  icon: DrawLayerIcon,
+  prerequisiteSteps: [
+    {
+      id: CREATE_DRAWN_FEATURES_INDEX_STEP_ID,
+      label: i18n.translate('xpack.maps.newVectorLayerWizard.indexFeatures', {
+        defaultMessage: 'Index features',
+      }),
+    },
+    {
+      id: ADD_DRAWN_FEATURES_TO_INDEX_STEP_ID,
+      label: i18n.translate('xpack.maps.newVectorLayerWizard.indexingFeatures', {
+        defaultMessage: 'Indexing features',
+      }),
+    },
+  ],
+  renderWizard: (renderWizardArguments: RenderWizardArguments) => {
+    return <NewVectorLayerEditor {...renderWizardArguments} />;
+  },
+  title: i18n.translate('xpack.maps.newVectorLayerWizard.title', {
+    defaultMessage: 'Draw new vector layer',
```

Comment:
How about changing the title to "Annotations" or "Blank documents layer"? I like "Blank documents layer" since it has better symmetry on how the layer will be used in the future. For example, user_a creates the new layer by selecting "Blank documents layer" and draws some features. Later, user_a wants to add those drawn features to a new map. What card will they use? How will they know to use "Documents" in the future? I think a name that description that tie this back to "Documents" will close the gap

Reply:
I see your point, but something makes me hesitate about "Draw documents". Those two words together are distracting to me, it sounds like the unrelated expression "Draw up a document" or a way of referring to art paper or something.

Maybe "Ingest drawn features as documents"?

Reply:
@gchaps Any suggestions for a good layer name?

Reply:
How about just "Draw"?

Reply:
Ingest is difficult for localization, so I don't recommend that.  I hesitate about "Draw documents" too.  "Draw" alone, is too vague.  It needs to have something after it. 

> Draw points and shapes and save in Elasticsearch".  

What does "save in Elasticsearch" mean? Index?

> Using the editor on the left side of the map, draw and edit the points and shapes to be indexed and added to the map.

Is it important to say "indexed and added to the map" here.  Or can it be something like this because the modal also prompts the user to give the index a name:

> Use the editor on the left to draw your points and shapes, then give your index a name. 
> Use the editor on the left to draw your points and shapes. 

Or, instead of "on the left" use an arrow.

> <- Use the drawing editor to create your points and shapes, then give your index a name.

Reply:
+1 to just calling it "Draw". The description provides enough details if users need more

Reply:
Proposal:

+1 to **Draw** as the title on the card

> Draw points, lines, and polygons to Elasticsearch

Draw points, lines, and polygons to create or edit documents.

> Draw vector shapes 

Draw shapes

> Using the editor on the left side of the map, draw and edit the points and shapes to be indexed and added to the map.

Use the drawing tools on the map to draw and edit shapes. All data is written to an index and includes a user name and time stamp.

Reply:
Thanks for the suggestions! I kind of picked from each of the suggestions, here's what I ended up with:

![image](https://user-images.githubusercontent.com/1728976/115468235-ca242c00-a200-11eb-87ea-a0a709bf12e0.png)

![image](https://user-images.githubusercontent.com/1728976/115468277-da3c0b80-a200-11eb-8ebd-5b22740e5336.png)

Due to flow considerations, this second wizard screen is going to change quite a bit in ~2 PRs or so when toolbar functionality and this wizard are merged together. I think the more succinct messaging everyone came up with here will work pretty well regardless. We can certainly iterate on it if needed though!